src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/machines/InfernalFarm.java

```diff
@@ -60,6 +60,20 @@ public Recipe getRecipe() {
         res.setIngredient('N', Material.NETHER_BRICK);
         return res;
     }
+    
+    @Override
+    public void onBlockRegistered(Location location, boolean isPlacing) {
+        int range = RADIUS / 2;
+        Block block = location.getBlock();
+
+        for (int x = -range; x <= range; x++) {
+            for (int z = -range; z <= range; z++) {
+                blocks.add(block.getRelative(x, 2, z));
+            }
+        }
+
+        super.onBlockRegistered(location, isPlacing);
+    }
```

Comment:
This is just horrible and redundant, by calling the super method you add all blocks on the same y level yet again.
And this is very redundant as it only deviates from the super method by a single parameter.

Reply:
Also the build is failing too...